packages/entity-browser/src/util/api.spec.js

```diff
@@ -42,45 +42,205 @@ describe('entity-browser', () => {
+            expect(res[0]).to.have.property('relGender')
+            expect(res[0].firstname).to.eql({type: 'string', value: 'Jon'})
+            expect(res[0].relGender).to.eql({type: 'string', value: 'Male'})
+            expect(res[0].titles).to.eql({type: 'string', value: 'Dr., Bundesrat'})
+
+            expect(res[1]).to.have.property('firstname')
+            expect(res[1]).to.have.property('relGender')
+            expect(res[1].firstname).to.eql({type: 'string', value: 'Klaus'})
+            expect(res[1].titles).to.eql({type: 'string', value: 'Dr., Prof'})
           })
         })
       })
 
       describe('fetchRecordCount', () => {
-        it('should call fetch and return corect amount', done => {
+        it('should call fetch and return correct amount', () => {
           fetchMock.get('*', {count: 99})
-          api.fetchRecordCount('User').then(result => {
+          return api.fetchRecordCount('User').then(result => {
             expect(result).to.be.eql(99)
+
             expect(fetchMock.calls().matched).to.have.length(1)
             const lastCallUrl = fetchMock.lastCall()[0]
             expect(lastCallUrl).to.eql('/nice2/rest/entities/User/count')
-            done()
           })
         })
       })
 
-      describe('fetchSearchForm', () => {
-        it('should call fetch', done => {
-          fetchMock.reset()
-          fetchMock.get('*', {data: [{fields: {a: 'a'}}]})
+      describe('fetchColumnDefinition', () => {
+        it('should call fetch and transform form to ColumnDefinition', () => {
+          const fetchResult = {
+            form: {
+              children: [{
+                name: 'table',
+                children: [
+                  {
+                    displayType: 'EDITABLE',
+                    label: 'label1',
+                    children: [{name: 'name1', type: 'type', displayType: 'EDITABLE', label: 'label'}]
+                  }, {
+                    displayType: 'HIDDEN',
+                    label: 'label2',
+                    children: [{name: 'name2', type: 'type', displayType: 'HIDDEN', label: 'label'}]
+                  }, {
+                    displayType: 'EDITABLE',
+                    label: 'label3',
+                    children: [{name: 'custom:name3', type: 'type', displayType: 'EDITABLE', label: 'label'}]
+                  }, {
+                    displayType: 'EDITABLE',
+                    label: 'label4',
+                    children: [{
+                      name: 'name4',
+                      type: 'ch.tocco.nice2.model.form.components.action.Action',
+                      displayType: 'EDITABLE',
+                      label: 'label'
+                    }]
+                  }, {
+                    displayType: 'EDITABLE',
+                    label: 'label5',
+                    children: [{name: 'name5', type: 'type', displayType: 'EDITABLE', label: 'label'}]
+                  }
+                ]
+              }
+              ]
```

Comment:
I would combine the brackets on column 192 and 193 like it is combined on line 162.